src/MiNET/MiNET/Scoreboards/ScoreboardDisplaySlot.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace MiNET.Scoreboards
+{
+
+    public enum ScoreboardDisplaySlot
+    {
+        List,
+        Sidebar,
+        BellowName
```

Comment:
Typo. Should be `BelowName`

Reply:
don't forget to shout as loud as you can